.travis.yml

```diff
@@ -9,9 +9,10 @@ before_install:
     - sudo apt-get install python3-dev
```

Comment:
that should not be the case in travis context:
https://github.com/zalando/ghe-backup/blob/adaptBackupSchedule/.travis.yml#L4